build/Dockerfile.ci

```diff
@@ -43,35 +43,35 @@ COPY tools.go tools.go
 RUN make build
 
 # Build WMCB
-WORKDIR /build/windows-machine-config-bootstrapper/
+WORKDIR /build/windows-machine-config-operator/windows-machine-config-bootstrapper/
 COPY windows-machine-config-bootstrapper/ .
 RUN make build
 
 # Build hybrid-overlay
-WORKDIR /build/ovn-kubernetes/
+WORKDIR /build/windows-machine-config-operator/ovn-kubernetes/
 COPY ovn-kubernetes/ .
-WORKDIR /build/ovn-kubernetes/go-controller/
+WORKDIR /build/windows-machine-config-operator/ovn-kubernetes/go-controller/
 RUN make windows
 
 # Build promu utility tool, needed to build the windows_exporter.exe metrics binary
-WORKDIR /build/promu/
+WORKDIR /build/windows-machine-config-operator/promu/
 COPY promu/ .
 RUN make build
 
 # Build windows_exporter
-WORKDIR /build/windows_exporter/
+WORKDIR /build/windows-machine-config-operator/windows_exporter/
 COPY windows_exporter/ .
-RUN GOOS=windows /build/promu/promu build -v
+RUN GOOS=windows /build/windows-machine-config-operator/promu/prometheus-promu build -v
 
 # Build Kubernetes node binaries
-WORKDIR /build/kubernetes/
+WORKDIR /build/windows-machine-config-operator/kubernetes/
 COPY kubernetes/ .
 ENV KUBE_BUILD_PLATFORMS windows/amd64
 RUN make WHAT=cmd/kubelet
 RUN make WHAT=cmd/kube-proxy
 
 # Build CNI plugins
-WORKDIR /build/containernetworking-plugins/
+WORKDIR /build/windows-machine-config-operator/containernetworking-plugins/
 COPY containernetworking-plugins/ .
 ENV CGO_ENABLED=0
 RUN ./build_windows.sh
```

Comment:
since we're copying the entire `/build/windows-machine-config-operator` over to the operator stage later, but don't need the submodules there, we could:
```
RUN rm -rf /build/windows-machine-config-operator/{containernetworking-plugins,kubernetes,ovn-kubernetes,promu,windows-machine-config-bootstrapper,windows_exporter}
```

Reply:
@LorbusChris are you sure this is safe to do? CI images typically has everything in the repo.

Reply:
oh yeah, you're right. Instead of removing it here, we should probably more selectively copy things over here:
https://github.com/openshift/windows-machine-config-operator/pull/168/files#diff-1fdb137609a5f55420f6dc837ca647db05cc80086d73dec390e696c97ed622dbL166 (exclude the submodules there)

Reply:
@LorbusChris the link does not work anymore, so I am not sure what you are suggesting.

Reply:
Moreover I thought the CI infra requires the full repo to present in the test image.

Reply:
I was referring to this: https://github.com/openshift/windows-machine-config-operator/blob/master/build/Dockerfile.ci#L166
Right now (without this PR) this doesn't include the submodules, but only these dirs:
https://github.com/openshift/windows-machine-config-operator/blob/master/build/Dockerfile.ci#L29-L42